plugin/storage/integration/integration.go

```diff
@@ -126,7 +130,8 @@ func (s *StorageIntegration) testGetServices(t *testing.T) {
 
 	var actual []string
 	found := s.waitForCondition(t, func(t *testing.T) bool {
-		actual, err := s.SpanReader.GetServices(context.Background())
+		var err error
+		actual, err = s.SpanReader.GetServices(context.Background())
```

Comment:
unnecessary change

Reply:
adding the new variable named `err` is required because  if we don't do that then a new `actual` variable get's Initialized at L134 and if the `GetServices` method fails we get the empty slice for the actual value as you can see below.

```
    integration.go:139: 
        	Error Trace:	/home/infracloud/go/src/github.com/jaeger/plugin/storage/integration/integration.go:139
        	Error:      	Should be true
        	Test:       	TestMemoryStorage/GetServices
    integration.go:140: 	 Expected: [example-service-1 example-service-2 example-service-3123]
    integration.go:141: 	 Actual  : []

```